### DISCUSSION TITLE
Committing to a forked repo results in an error

### DISCUSSION THREAD

**Discussion by Johannes5** [gilbarbara/react-spotify-web-playback#140] [General]
I've forked https://github.com/gilbarbara/react-spotify-web-playback.git
by creating a cloned Project in Webstorm and then publishing

these are the upstream repos listed:
github	https://github.com/Johannes5/react-spotify-web-playback.git (fetch)
github	https://github.com/Johannes5/react-spotify-web-playback.git (push)
origin	https://github.com/gilbarbara/react-spotify-web-playback.git (fetch)
origin	https://github.com/gilbarbara/react-spotify-web-playback.git (push)

The first commit I attempted to make resulted in this message:
✔ Repo is up-to-date!
.husky/pre-commit: line 4: lint-staged: command not found
husky - pre-commit hook exited with code 127 (error)
husky - command not found in PATH=/usr/local/Cellar/git/2.30.1/libexec/gi...

I was hoping you might know how to solve this.
I tried running the "prepare" script. I'm not used to husky or lint-staged.

Thanks in advance.

PS: the reason I forked it was so I have more freedom to style the the player, eg by changing stuff inside of Player.tsx

**Comment by gilbarbara** [comment]
Hey @Johannes5 

To clarify, you don't have a fork but a clone. So your repository is disconnected from this one, and you won't get updates...
I've fixed the problem with husky, but you won't get it automatically.
I suggest you remove the clone and fork it properly.
